src/Drupal/Commands/core/LocaleCommands.php

```diff
@@ -144,76 +145,121 @@ public function update($options = ['langcodes' => self::REQ])
      *   Export the Dutch non-customized translations.
      * @usage drush locale:export --langcode=nl --types=customized,not-customized > nl.po
      *   Export the Dutch customized and not customized translations.
-     * @usage drush locale:export > drupal.pot
+     * @usage drush locale:export --template > drupal.pot
      *   Export the template file to translate.
      * @aliases locale-export
      * @see \Drupal\locale\Form\ExportForm::submitForm
      *
      * @throws \Exception
      */
-    public function export($options = ['langcode' => self::OPT, 'types' => self::OPT])
+    public function export($options = ['template' => false, 'langcode' => self::OPT, 'types' => self::OPT])
```

Comment:
Minor: Order by importance / priority / frequency of use. Move 'template' to the end.